docs/01-app/03-api-reference/05-config/01-next-config-js/turbo.mdx

```diff
@@ -45,14 +45,18 @@ module.exports = nextConfig
 
 The following options are available for the `turbo` configuration:
 
-| Option              | Description                                                               |
-| ------------------- | ------------------------------------------------------------------------- |
-| `rules`             | List of unsupported webpack loaders to apply when running with Turbopack. |
-| `resolveAlias`      | Map aliased imports to modules to load in their place.                    |
-| `resolveExtensions` | List of extensions to resolve when importing files.                       |
-| `moduleIdStrategy`  | Assign module IDs                                                         |
-| `treeShaking`       | Enable tree shaking for the turbopack dev server and build.               |
-| `memoryLimit`       | A target memory limit for turbo, in bytes.                                |
+| Option                    | Type                                            | Description                                                                |
+| ------------------------- | ----------------------------------------------- | -------------------------------------------------------------------------- |
+| `loaders`                 | `Record<string, TurboLoaderItem[]>`             | List of webpack loaders to apply when running with Turbopack.              |
```

Comment:
That doesn't exist anymore. It has been removed in favor of `rules`

Reply:
Removed 👍🏼